assignments/array-methods.js

```diff
@@ -89,12 +89,12 @@ console.log(ticketPriceTotal);
 
 // Problem 1
 let namesArr = [];
-runners.forEach (item => namesArr.push(('${item.last_name}, ${item.first_name}')));
-namesArr.sort(function(a, b) {
+runners.forEach (item => namesArr.push((`${item.last_name}, ${item.first_name}`)));
+namesArr.sort((a, b) => {
```

Comment:
This function you're running inside of sort isn't doing anything. 

This produces literally the same output:
```suggestion
namesArr.sort()
```

What are you trying to do here? Maybe I can help you achieve the desired output.